### PR TITLE
fix windows build

### PR DESCRIPTION
**Pull request by m4r71n**
was broken with efff5a3daeb0571bb3fe8345b72d3f77b79d0066

### CODE REVIEW

File: src/app/app.cpp [hunk 1]
```diff
@@ -35,6 +35,7 @@
 
 #ifndef COPYQ_TRANSLATION_PREFIX
 #   ifdef Q_OS_WIN
+#       include <QApplication>
 #       define COPYQ_TRANSLATION_PREFIX QApplication::applicationDirPath() + "/translations"
```
Comment (hluk) [inline src/app/app.cpp]: Can you rather change `QApplication` on this line to `QCoreApplication`?

Comment (m4r71n) [inline src/app/app.cpp]: I'll try it but not before evening

Comment (m4r71n) [inline src/app/app.cpp]: No unfortunately not. Btw. `QCoreApplication` is already included in [line 29](https://github.com/hluk/CopyQ/pull/200#discussion-diff-11055958L29)

Comment (hluk) [inline src/app/app.cpp]: I actually meant using `QCoreApplication::applicationDirPath()` on the line below so the header `QApplication` doesn't need to be included here.

Comment (m4r71n) [inline src/app/app.cpp]: Sorry that was a misunderstanding from me. I've updated the change.

Comment (hluk) [inline src/app/app.cpp]: No problem, thanks.

**Pull request merged**